Add objectives + timelines (#6)
mlopscommunity/community-survey: README.md

Our goal is to create a survey that helps us, data science & MLOps practitioners, learn more about our community and (in the future) track how our community changes year-over-year.  Said differently - this is a survey for the community, by the community.  We designed this survey to gain an accurate understanding of we, the professionals designing, building and deploying **production** ML systems, individually and collectively are, the types of data and models we implement, the tools we use, and the challenges we face.  Specifically, this is NOT a vendor-led survey that pushes forward a specific perspective that advantages a vendor or group of vendors.

This document provides an outline of the topics we want to understand from our community.  It is intended to spark discussion and drive alignment on set of topics and learning goals within each topic.

# Objectives & goals

Specifically, our goals are:

> * Develop a ~15 minute survey that provides a neutral, vendor-agnostic view of how practitioners apply data science

Reviewer: is that really what we care about? i thought it was more interesting to know about the ML setups or how they are doing MLOps? 

> how practitioners apply data science

Author: I was trying to summarize all of that as “apply data science”

Maybe “implement data science in the real world”?

> * Develop a ~15 minute survey that provides a neutral, vendor-agnostic view of how practitioners apply data science

Author: ```suggestion
* Develop a ~10 minute survey that provides a neutral, vendor-agnostic view of how practitioners apply data science
```

> * Publish an annual report on the `State of Data Science` based on the survey results

Reviewer 2: Should we have it as 'State of MLOps'? (I realise that Data Science is more clickable - so it might be worth using it in a subheading, but is it the main objective?)

Author: I think "State of MLOps" is more accurate, but I agree with the click bait attractiveness of "data science"

Let's call is "State of MLOps" internally here since its more accurate, then we can optimize the SEO of the blog seperatly?

Author: ```suggestion
* Publish an annual report on the `State of MLOps` based on the survey results
```

* Survey 1,000 data science practitioners who are representative of the broader community (per year)

> * Provide open-source access to the (anonymized) data

Author: Adding a suggestion from @andreeamun 

```suggestion
* Provide open-source access to the (anonymized) data
* Learn about our community and our MLOps practices (and in the future, track this over time)
```

# Objectives & goals

Specifically, our goals are:

> * Develop a ~10 minute survey that provides a neutral, vendor-agnostic view of _who_ is in our community and _why and how_ MLOps is applied.

Author: Updated  with @dpbrinkm feedback

Reviewer 2: looks good to me

Reviewer: I like this

# Timeline

We are working backwards from an April 2023 release date for the report.  We are hopeful that our proposed KubeCon talk on this survey will be accepted.

- November 7 - 18: Engage with various community members to evangelize idea, identify contributors, get feedback, etc
- November 22: Finalized list of topics & learning goals
- December 9: First draft of survey text written
- December 10 - 22: Collect feedback on the draft survey text
- December 23 - January 5: Holiday break

> - January 13: Survey release candidate #1 available as a web link

Reviewer: should we say how we want to distribute it? what channels and with the help of who?

Author: Yes, Andreea was going to work on a plan for this.
